test/evaluation_test.py

```diff
@@ -0,0 +1,36 @@
+
+def evaluation_test(uid, upwd):
+    auth = (uid, upwd)
+    base = 'http://localhost:8000/ttrs/evaluations/'
+
+    import json, requests
+    from ttrs.models import Lecture
+
+    print('\ntesting '+'\033[1m'+base+'\033[0m'+'...')
+    for i in range(1,4):
+        data = {}
+        data['rate'] = 5
+        data['comment'] = 'Wonderful lecture!'
+        data['lecture'] = Lecture.objects.all()[i].id
+
+        print('\nCreating Evaluation {}...'.format(i))
+        res = requests.post(base, auth=auth, data=data)
+
+        if res.status_code == 201:
+            print('Successfully created Evaluation instance.')
+            print(res.text)
+        else:
+            print('Error creating Evaluation instance.')
+            print(res.text)
+
+    print('\n'+'\033[1m'+'testing invalid data...'+'\033[0m')
+    data = {}
+    data['rate'] = 99
+    data['comment'] = ''
+    data['lecture'] = 0
+
+    res = requests.post(base, auth=auth, data=data)
+    print(res)
+    errors = json.loads(res.text)
+    for error in errors:
+        print(error, errors.get(error))
```

Comment:
I think we should check status code.

Reply:
And, when I run test, should I check error case in my eyes?
It would be better to let python to check the error messages and just print whether this test succeeded.

Reply:
print(res) prints out status code like below
![Uploading Screenshot from 2018-05-12 23-17-29.png…]()
Wouldn't it be enough?

Reply:
At now, we have to check each line of log that test program prints.
It is better than when we input some arbitrary data to the server and waits response.
But, I think there is room for more automation.
I think it is ideal that the tester don't need to check many lines of log, just can finish test by checking one line of the log; "test succeeded" or "test failed: at ~~".
If you think this is hard to implement, just leave it, I'll make later.

Reply:
Not a big deal. I can just comment out _print_s.

Reply:
Isn't there no need to comment out 'print's?
I think it is enough to let the tester know whether the test is succeeded or not at the end of the test.